core-web/src/main/java/org/silverpeas/core/webapi/admin/delegate/AdminPersonalWebDelegate.java

```diff
@@ -27,6 +27,7 @@
 import java.util.LinkedHashMap;
 import java.util.Map;
 
+import org.silverpeas.core.SilverpeasExceptionMessages;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.silverpeas.core.SilverpeasExceptionMessages'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)